description.md

```diff
@@ -21,6 +21,41 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Tutorials
 
+Tutorials are pieces of content directed toward the reproduction or verification of a practical result.
+Tutorials are step-by-step presentations of a topic, and are supposed to be read by participants individually.
+They can be used as part of a practical session, with assistance provided to further explaining certain items, or as unassisted self-study.
+
+Tutorials consist of:
+
+- text: created following the contents of the [`Reading` section](#reading)
+- media: created following the contents of the [`Media` section](#section)
+- practice files: source code, diagrams or any digital information to be used by participants
```

Comment:
```suggestion
- practice files: source code, diagrams or any other file type that is to be modified by participants as part of the  tutorial activity
```

Reply:
I think the above clarification is needed as to not confuse practice files with media or drills.

Reply:
Agreed.